### PR TITLE
doc(release) Release notes 24.10.0 Modules

### PR DESCRIPTION
**Pull request by pkippes**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 1]
```diff
@@ -1,145 +1,157 @@
...
-Release date: `April 30, 2024`
+Release date: `October 30, 2024`
 
-<details>
-  <summary>Compatibility</summary>
+<details open>
+  <summary>Enhancements</summary>
 
-- Compatibility with other 24.04 components.
+- [MBI] Improved the starting of CBIS by using systemd configuration.
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:77]:
```suggestion
- [MBI] The CBIS service is now started using systemd's configuration.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 2]
```diff
@@ -15,130 +15,146 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
-Release date: `April 30, 2024`
+Release date: `October 30, 2024`
 
-<details>
-  <summary>Compatibility</summary>
+<details open>
+  <summary>Enhancements</summary>
 
-- Compatibility with other 24.04 components.
+- [MBI] Improved the starting of CBIS by using systemd configuration.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:80]:
```suggestion
- [MBI] The CBIS service is now started using systemd's configuration.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 3]
```diff
@@ -15,130 +15,145 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
+- [Viewer] You can now display the status as background color for a resource in weather style.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Server] Fixed a synchronization issue that could cause resources to disappear from a map.
+- [Server] Fixed concurrent access issues causing exceptions in the viewer.
+- [Server] Fixed Servicegroup status when configured using Service Templates or Hostgroup Services.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:53]:
```suggestion
- [Server] When using Service Templates or Hostgroup Services, the status of Service groups is now displayed correctly.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 4]
```diff
@@ -1,145 +1,156 @@
...
 > It is very important when you update your system to refer to this section in order to learn about behavior changes or
-> major changes that have been made on this version. This will inform you about the impact of installing these
+> major changes that have been made on this version. This will let you know the impact of the installation of these
 > versions on the features you use or the specific developments that you have built on your platform (modules,
 > widgets, plugins).
 
 If you have feature requests or want to report a bug, please contact support.
 
-> Read more about version 24.04 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-04/).
+Read more about version 24.10 in our [blog post](https://www.centreon.com/en/blog/centreon-fall24-whats-new-in-the-24-10-software-version/).
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:15]:
```suggestion
Read more about version 24.10 in our [blog post](https://www.centreon.com/centreon-announces-its-latest-software-release-centreon-24-10/).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 5]
```diff
@@ -15,130 +15,156 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
+- Increased the number of available fonts and made Roboto the default one.
 
-## Centreon BAM
+Viewer:
+- You can now display a host name for a service.
+- You can now display the status as background color for a resource in weather style.
 
-### 24.04.0
+GeoViews:
+- Redesigned the edit modal.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:55]:
```suggestion
- Redesigned the edit window.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 6]
```diff
@@ -15,130 +15,157 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 
-Release date: `April 30, 2024`
+Release date: `October 30, 2024`
 
+<details open>
+  <summary>Map Legacy retirement</summary>
+
+- **MAP Legacy is no longer available.**
+- If you are still using Map Legacy and want to know more about how to migrate to MAP, read [MAP Legacy end of life](https://docs.centreon.com/fr/docs/graph-views/map-legacy-eol/).
+</details>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:33]:
```suggestion

</details>
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 7]
```diff
@@ -15,130 +15,157 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 <details open>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:34]:
```suggestion

<details open>
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 8]
```diff
@@ -1,145 +1,168 @@
...
+
+Release date: `October 30, 2024`
+
+<details open>
+  <summary>Map Legacy retirement</summary>
 
-Release date: `April 30, 2024`
+- **MAP Legacy is no longer available.**
+- If you are still using Map Legacy and want to know more about how to migrate to MAP, read [MAP Legacy end of life](https://docs.centreon.com/docs/graph-views/map-legacy-eol/).
+</details>
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:30]:
```suggestion

</details>
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by rsadik-ext — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 9]
```diff
@@ -15,130 +15,159 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 
-Release date: `April 30, 2024`
+Release date: `October 30, 2024`
 
 <details open>
-  <summary>Enhancements</summary>
+  <summary>Map Legacy retirement</summary>
 
-- To facilitate a smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.
+- **MAP Legacy is no longer available.**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:31]:
```suggestion
> **MAP Legacy is no longer available.**
```

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 10]
```diff
@@ -7,139 +7,168 @@ import ExpandCollapseAll from '@site/src/components/ExpandCollapseAll';
...
 
-Release date: `April 30, 2024`
+Release date: `October 30, 2024`
 
 <details open>
-  <summary>Enhancements</summary>
+  <summary>Map Legacy retirement</summary>
 
-- To facilitate a smooth transition from MAP Legacy to MAP, we have made sure that features and enhancements already available in previous versions are fully compatible.
+- **MAP Legacy is no longer available.**
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:32]:
```suggestion
> **MAP Legacy is no longer available.**
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 11]
```diff
@@ -15,130 +15,161 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 
-New playlist feature: dashboard playlists allow you to display several dashboards sequentially without any interaction. Ideal for broadcast on a TV screen. 
-
-You can now:
-- create dashboard playlists.
-- display the list of dashboard playlists.
-- share playlists.
-- display the list of playlist shares.
-- revoke the sharing of a playlist to a contact group.
+Playlists of Dashboards
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:145]:
```suggestion
**Dashboard playlists**
```

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 12]
```diff
@@ -7,139 +7,168 @@ import ExpandCollapseAll from '@site/src/components/ExpandCollapseAll';
...
 
-New playlist feature: dashboard playlists allow you to display several dashboards sequentially without any interaction. Ideal for broadcast on a TV screen. 
-	
-You can now:
-- create dashboard playlists.
-- display the list of dashboard playlists.
-- share playlists.
-- display the list of playlist shares.
-- revoke the sharing of a playlist to a contact group.
+- [Dashboards] A new PATCH endpoint has been added for updating playlists.
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:146]:
```suggestion
**Dashboard playlists**
- Added unauthenticated playlists for TV projection.
- A new PATCH endpoint has been added for updating playlists.
```

File: versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 13]
```diff
@@ -7,139 +7,172 @@ import ExpandCollapseAll from '@site/src/components/ExpandCollapseAll';
...
 
-- Compatibility with other 24.04 components.
+Release date: `October 30, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+**Dashboards**
+- A new "Business Activity Diagram" widget is now available.
+- Added the handling of Business Activities and Business Views in the Status Grid widget.
```
Comment (cg-tw) [inline versioned_docs/version-24.10/releases/centreon-commercial-extensions.mdx:82]:
```suggestion
- Business Activities and Business Views can now be displayed in the Status Grid widget.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx [hunk 14]
```diff
@@ -15,130 +15,163 @@ Vous trouverez dans ce chapitre tout ce qui concerne les **extensions commercial
...
 
-- Compatibility with other 24.04 components.
+Release date: `October 30, 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+**Dashboards**
+- A new "Business Activity Diagram" widget is now available.
+- Added the handling of Business Activities and Business Views in the Status Grid widget.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-commercial-extensions.mdx:81]:
```suggestion
- Business Activities and Business Views can now be displayed in the Status Grid widget.
```

**Review by cg-tw — APPROVED** [review]

**Review by rbignotticentreon — COMMENTED** [review]
Parties MAP et DASHBOARDS => OK

**Review by rbignotticentreon — APPROVED** [review]
Parties MAP et DASHBOARDS => OK

**Review by rsadik-ext — APPROVED** [review]

**Pull request merged**